manage_orderbook.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+
+from binance.spot import Spot as Client
+from binance.websocket.spot.websocket_client import SpotWebsocketClient
+from binance.lib.utils import config_logging
+import os
+import logging
+import json
+import asyncio
+
+"""
+Creates, manages a local orderbook and prints out its best prices.
+Based on https://binance-docs.github.io/apidocs/spot/en/#diff-depth-stream
+
+Instructions:
+0.pip install binance-connector-python
+1.Set your own environment variables (BINANCE_TESTNET_API_KEY, BINANCE_TESTNET_SECRET_KEY, BINANCE_SYMBOL,
+                                      BINANCE_BASE_URL, BINANCE_STREAM_URL):
+  Production:
+  BINANCE_BASE_URL='https://api.binance.com', BINANCE_STREAM_URL='wss://stream.binance.com:9443/ws'
```

Comment:
We can provide url in the configure file or just hardcode in this script. Let the client to easily to use, e.g.

WORK_ENV=test BINANCE_API_KEY=xxx BINANCE_API_SECRET=xxxx SYMBOL=xxxx python manage-orderbook.py